cypress/objects/loginPage.js

```diff
@@ -3,7 +3,7 @@ class LoginPage {
       cy.visit('/')
 }
```

Comment:
Starting from here right down to the second-to-last bracket, all code should be indented to the right by one tab. That would be the correct way to format your code :)